packages/cheminfo-build/bin/cheminfo-build.js

```diff
@@ -9,7 +9,7 @@ const rollup = require('rollup');
 const babel = require('rollup-plugin-babel');
 const commonjs = require('@rollup/plugin-commonjs');
 const json = require('@rollup/plugin-json');
-const resolve = require('@rollup/plugin-node-resolve');
+const nodeResolve = require('@rollup/plugin-node-resolve');
```

Comment:
```suggestion
const { nodeResolve } = require('@rollup/plugin-node-resolve');
```

I think it needs to be like this. Can you try to edit your local copy to see if it works?